### PR TITLE
DS-957 Android documentation for Pill

### PR DESCRIPTION
**Pull request by willmee**

### CODE REVIEW

**Comment by danoc** [comment]
Looks good to me! @lasya-tack, could you also take a look? I think @mallikapotter is OOO.

File: www/src/pages/components/pill/android/index.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+import { graphql } from 'gatsby';
+import { ComponentHeader } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
+
+### Usage
+
+Pills are compact badges that help to explain or draw attention to a certain area of the UI. Pills include one or both of text and an icon (which at the start of the text if both are present).
```
[the remark this answers is not in the document]
Comment (willmee) [reply www/src/pages/components/pill/android/index.mdx:13]: Done.

File: www/src/pages/components/pill/android/index.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+For example:
+
+```xml
+    <com.thumbtack.thumbprint.views.pill.ThumbprintPill
+        android:id="@+id/indigoPill"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:pillColor="indigo"
+        app:pillText="Indigo Pill" />
```
[the remark this answers is not in the document]
Comment (willmee) [reply www/src/pages/components/pill/android/index.mdx:51]: Done.

File: www/src/pages/components/pill/android/index.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Pill uses Thumbprint Title7 text (14sp)
+
+### Layout XML
+
+Pill instances are configured in layout XML, using the custom attributes
+
+-   `pillColor` one of the seven colors determining the style
+-   `pillText` text to use
+-   `pillIcon` an optional icon
```
[the remark this answers is not in the document]
Comment (willmee) [reply www/src/pages/components/pill/android/index.mdx:41]: Done.

**Review by kevinmbeaulieu — DISMISSED** [review]

File: www/src/pages/components/fab/ios/index.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+#### TextFloatingActionButton Secondary
+
+![Secondary text fab](/img/ios/text_fab_secondary.png)
+
+#### IconFloatingActionButton Primary
+
+![Primary icon fab](/img/ios/icon_fab_primary.png)
+
+#### IconFloatingActionButton Secondary
```
Comment (lasya-tack) [inline www/src/pages/components/fab/ios/index.mdx:35]: I believe icon fab is meant to support only primary style. https://www.figma.com/file/YrpLNWoOwvEADbyFBWzuKv0V/Thumbprint-Global?node-id=1258%3A90

But I see web supports both as well. Maybe we should add support for it on android too? 🤷🏼‍♀️  @mallikapotter

**Review by lasya-tack — APPROVED** [review]
Just one comment re: secondary IconFab but not blocking.

File: www/src/pages/components/fab/ios/index.mdx (hunk 4 above)
Comment (kevinmbeaulieu) [reply www/src/pages/components/fab/ios/index.mdx:35]: iOS supports both too- I think that figma might just not be complete. Does Android currently support the secondary style or does it need to be updated before it'd work?

Comment (lasya-tack) [reply www/src/pages/components/fab/ios/index.mdx:35]: Android only supports primary style for IconFab based on the requirements here - https://docs.google.com/document/d/1yms-6TGSliYF0I3nyX5oxH-GOWORRgXyxJseVh0LJuM/edit
I'll let Mallika weigh in here.

**Comment by willmee** [comment]
I'm not sure why the FAB change is part of this pull request. The only text file I intended to be here is for the Pill component

www/src/pages/components/pill/android/index.mdx

This is the first time I'm doing a pull request outside of Gerrit: I think I might have  corrupted the request?

**Comment by danoc** [comment]
> I'm not sure why the FAB change is part of this pull request. The only text file I intended to be here is for the Pill component
> 
> www/src/pages/components/pill/android/index.mdx
> 
> This is the first time I'm doing a pull request outside of Gerrit: I think I might have corrupted the request?

Not too sure, but I've updated the PR to only include the Pill files.

**Review by lasya-tack — APPROVED** [review]

**Comment by danoc** [comment]
Thanks for the contribution, @willmee! The changes will be live in ~5 mins.

**Pull request merged**